guides/how_sprockets_works.md

```diff
@@ -392,24 +412,18 @@ Another example that we have in the shop file application is we actually have a
 //= link_tree ../images .png
 ```
 
-This is the real code. It's just a `call` method that actually gets the input that is the SVG source code and ask the image to change to generate a PNG file and we return that PNG file in the data.
+This is the real code. It's just a `call` method that gets the input that is the SVG source code and uses RMagick to generate the PNG file which is returned as a binary blob under the `data` Hash key.
 
 ```ruby
-require ‘rmagick’
+require 'rmagick'
 
 class SvgTransformer
   def self.call(input)
-    image_list = Magick::Image.from_blob(input[:data]) { self.format = ‘SVG’ }
+    image_list = Magick::Image.from_blob(input[:data]) { self.format = 'SVG' }
     image = image_list.first
-    image.format = ‘PNG’
+    image.format = 'PNG'
 
     { data: image.to_blob }
   end
 end
 ```
-
-So, my effort in this talk is that I know that Sprockets is used in many Rails applications right now, but many users don't even know how it exists. Many users don't know how it works. Even I didn't know how it works two years ago, so it's important for you to try to understand your tools. Documenting your understandings, doing talks or writing documentation for these tools, and share with the community. So, we are right now in the effort to save Sprockets so you can see more about that tomorrow in Richard's talk that's called Saving Sprockets, of course. And that's it. 
```

Comment:
I removed this section because it spoke directly to the talk and didn't seem to be directly related to how Sprokets works.